server/routes/api.js

```diff
@@ -199,17 +199,113 @@ router.post('/event', passport.authenticate('jwt', {session: false}), async (req
 
   // Add the creator as an attendee in the back-end
   const member_event = new Member_Event({
-    date: new Date(),
+    date: [new Date()],
     paid: false,
     member: req.body.creatorId,
-    event: event._id
+    event: event._id,
+    tickets: 0
   });
   member_event.save()
     .catch(err => {
       console.log(err);
   });
 });
 
+router.post('/ticket',passport.authenticate('jwt', {session: false}), async (req, res)=>{
+    try{
+        const { userId, eventId } = req.body;
+
+        const event = await Event.findById(eventId);
+        const user = await Member.findById(userId);
+
+        if(!event || !user){
+            return res.status(404).json({ error: 'User or Event not found' });
+    
+        }
+        const event_member = await Member_Event.findOne({
+            member: user._id,
+            event: event._id
+        });
+        if(event_member){
```

Comment:
Just because there is the event_member doesn't mean that the member has a ticket. if the event_member.paid is true, then they have bought a ticket.